compose.yml

```diff
@@ -24,15 +24,45 @@ services:
     depends_on:
       - bootstrap
       - etcd
+      - mix-node-0
+      - mix-node-1
+      - mix-node-2
     environment:
-      - DOCKER_REPLICAS=${DOCKER_COMPOSE_LIBP2P_REPLICAS:-1}
+      - LIBP2P_REPLICAS=${DOCKER_COMPOSE_LIBP2P_REPLICAS:-1}
```

Comment:
All looks good, but the `nomos-node-libp2p-node-1` container is sometimes not shutdown without any log.
```
nomos-node-libp2p-node-1 exited with code 1
```
I'm taking a look at it.